src/v2/Apps/Fair/FairApp.tsx

```diff
@@ -80,7 +80,7 @@ const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
           onClick={trackTabData(
             `${fairHref}/booths`,
             "Booths",
-            ContextModule.exhibitorsTab
+            "boothsTab" as ContextModule
```

Comment:
**suggestion / non-blocking**: Let's make sure to ticket this new context so we can swap `ContextModule.boothsTab` in here.

cc/ @louislecluse FYI ☝🏼

Reply:
@dblandin should I do something with it? Or leave it as it is?

Reply:
You can leave it as is for this PR! I'll follow up with another Jira ticket.